.github/workflows/ci-example-project.yaml

```diff
@@ -19,6 +19,5 @@ jobs:
         run: ./nix-shell.sh
       - name: Build all
         run: |
-          ./nix-shell.sh "--run \
-          cd example_project_bzl_4 \
-          bazel build //..."
+          ./nix-shell.sh "--run cd example_project_bzl_4" 
+          ./nix-shell.sh "--run bazel build //..."
```

Comment:
Wait, doesn't this lose context now? We're restarting the script, so we're relying on the staying where we were. I think I prefer the set of operations to be atomic within a spawned shell.

Reply:
Seconded.